sdk/servicebus/Azure.Messaging.ServiceBus/tests/SessionRecords/ServiceBusManagementClientLiveTests/SqlFilterParamsAsync.json

```diff
@@ -1,18 +1,18 @@
 {
   "Entries": [
     {
-      "RequestUri": "https://recordedtests.servicebus.windows.net/d23cafe1?api-version=2017-04",
```

Comment:
Why did we need to re-record all the tests?

Reply:
May not have been necessary;  I usually do a refresh when I make public API changes to the associated class.   In this case, the change was to the constructor, so it should have been non-impactful, but it's also a good forcing function to grab a new capture.

Reply:
In general, it is better to avoid doing a re-rerecording unless the tests fail for an expected reason. Otherwise we can miss unexpected changes to the behavior.